### PR TITLE
Feature/support symfony 5

### PR DESCRIPTION
**Pull request by AurimasVilys**

### CODE REVIEW

File: DependencyInjection/SyliusRbacExtension.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public function load(array $config, ContainerBuilder $container)
      */
     public function prepend(ContainerBuilder $container)
     {
-        $this->prependDoctrineCache($container);
+        $this->prependSyliusResource($container);
```
Comment (GrandLTU) [inline DependencyInjection/SyliusRbacExtension.php:67]: why `prependSyliusResource` is called twice?

File: DependencyInjection/SyliusRbacExtension.php [hunk 2]
```diff
@@ -73,16 +73,13 @@ public function prepend(ContainerBuilder $container)
      */
     private function prependDoctrineCache(ContainerBuilder $container)
```
Comment (GrandLTU) [inline DependencyInjection/SyliusRbacExtension.php]: `prependDoctrineCache` is left unused?

File: DependencyInjection/SyliusRbacExtension.php (hunk 1 above)
Comment (AurimasVilys) [reply DependencyInjection/SyliusRbacExtension.php:67]: Mistake done while updating file

File: DependencyInjection/SyliusRbacExtension.php (hunk 2 above)
Comment (AurimasVilys) [reply DependencyInjection/SyliusRbacExtension.php]: Yes, as the cache is configured in app/config.yml where previously the parameter was defined

**Pull request merged**